Readme.md

```diff
@@ -1,24 +1,25 @@
-# pino-multi-stream &nbsp;[![Build Status](https://travis-ci.org/pinojs/pino-multi-stream.svg?branch=master)](https://travis-ci.org/pinojs/pino-multi-stream)
+# pino-multi-stream &nbsp;[![Build Status](https://badgen.net/github/status/pinojs/pino-multi-stream/master/ci)](https://github.com/pinojs/pino-multi-stream/actions?query=workflow%3ACI)
```

Comment:
We're using https://shields.io/ in the core `pino` repo. I think it would be good to be consistent and use it here as well.

Reply:
Sure, will change here and in pino-pretty#101

Reply:
Done now!